### PR TITLE
Fix for #1624 iOS 14 send temeletry fix

### PR DESCRIPTION
**Pull request by jbobotek**

### CODE REVIEW

File: pal/ios-osx/tlsio_appleios.c [hunk 1]
```diff
@@ -521,7 +521,14 @@ static void dowork_poll_socket(TLS_IO_INSTANCE* tls_io_instance)
     CFStreamCreatePairWithSocketToHost(NULL, tls_io_instance->hostname, tls_io_instance->port, &tls_io_instance->sockRead, &tls_io_instance->sockWrite);
     if (tls_io_instance->sockRead != NULL && tls_io_instance->sockWrite != NULL)
     {
-        if (CFReadStreamSetProperty(tls_io_instance->sockRead, kCFStreamPropertySSLSettings, kCFStreamSocketSecurityLevelNegotiatedSSL))
+        
+        CFStringRef keys[1] = {kCFStreamPropertySocketSecurityLevel};
+        CFStringRef values[1] = {kCFStreamSocketSecurityLevelNegotiatedSSL};
+
+        CFDictionaryRef test = CFDictionaryCreate(NULL , (void *)keys , (void *)values , 1,  NULL , NULL);
```
Comment (danewalton) [inline pal/ios-osx/tlsio_appleios.c:528]: reason why this is named test?

Comment (jbobotek) [reply pal/ios-osx/tlsio_appleios.c:528]: No reason. change to tls_io_dictionary?

Comment (danewalton) [reply pal/ios-osx/tlsio_appleios.c:528]: Yea if this is going to be merged this should be updated. Is this PR testing a fix or looking to be merged. The title is a little confusing

Comment (jbobotek) [reply pal/ios-osx/tlsio_appleios.c]: Updated and title corrected.

**Review by danewalton — APPROVED** [review]

**Pull request merged**